dirty_cat/datasets/tests/test_fetching.py

```diff
@@ -392,9 +386,17 @@ def test_import_all_datasets(
         assert expected_return_value == returned_value
 
 
+def test_fetch_world_bank_data():
+    indicator_name = 'gdppc'
+    df = fetch_world_bank_indicator('NY.GDP.PCAP.CD', indicator_name).X
+    assert isinstance(df, pd.DataFrame)
+    assert df.columns[1] == indicator_name
+    assert fetch_world_bank_indicator('NY.GDP.PCAP.CD', indicator_name, load_dataframe=False).name == indicator_name
```

Comment:
So, technically, this works, but if the CI cannot reach the Worldbank's website, it will crash and fail the test (though the code might work).
You might want to check how other fetching tests are done (mocking, monkey-patching and just ignoring the test if there is an internet connection (though it's not the optimal solution, as pointed out in one of the test)).

Reply:
Thanks, very important. I think I managed to fix it.